.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,13 @@
+name: Auto approve
```

Comment:
Do you have a favorite package for testing this type of thing locally?

Reply:
nope. I test on github lol